src/app/(dashboard)/projects/page.tsx

```diff
@@ -0,0 +1,74 @@
+"use client";
+
+import BreadCrumb from "@/components/breadcrumbs";
+import { Button } from "@/components/ui/button";
+import { Heading } from "@/components/ui/heading";
+import { Separator } from "@/components/ui/separator";
+import axios from "axios";
+import { Plus } from "lucide-react";
+import { useRouter } from "next/navigation";
+import { useEffect, useState } from "react";
+import { Team } from "@prisma/client";
+import { TextGenerateEffect } from "@/components/ui/text-generate-effect";
+import { ProjectCard } from "@/components/projects/projectCard";
+
+const breadcrumbItems = [{ title: "Projects", link: "/dashboard/projects" }];
```

Comment:
We usually put this inside the export default function TeamsPage(). Maybe for consistency keep it the same as the rest?